src/main/java/spoon/reflect/code/CtComment.java

```diff
@@ -59,12 +60,21 @@
 	String LINE_SEPARATOR = "\n";
 
 	/**
-	 * Get the content of the comment
-	 * @return the content of the comment
+	 * Get the content of the comment.
+	 * The returned content is a parsed version of the raw content:
+	 *   - without the delimiters (/*, //)
+	 *   - without the intermediate prefix " * " in case of Javadoc API comments.
+	 *
+	 * @see #getRawContent() for the raw comment.
+	 * @return the parsed content of the comment
 	 */
-	@PropertyGetter(role = COMMENT_CONTENT)
+	@DerivedProperty
 	String getContent();
 
+	/** The exact string that appears in source code, with delimiters */
+	@PropertyGetter(role = COMMENT_CONTENT)
+	String getRawContent();
+
 	@PropertySetter(role = COMMENT_CONTENT)
 	<E extends CtComment> E setContent(String content);
```

Comment:
If you keep `getRawComment`, then you should provide `setRawContent` too.

Reply:
added doc

Reply:
added documentation